user_service/apis/handlers/user.go

```diff
@@ -38,31 +40,44 @@ func (uh *UserHandlers) HandleGetProfile(ctx *gin.Context) {
 }
 
 func (uh *UserHandlers) HandleLogin(ctx *gin.Context) {
-	var requestBody models.LoginRequestBody
+	var requestBody models.LoginRequest
 
 	if err := ctx.ShouldBindJSON(&requestBody); err != nil {
 		ctx.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
 		return
 	}
-	message := uh.userService.Login(requestBody)
-
+	message, err := uh.userService.Login(requestBody)
+	if err != nil {
+		ctx.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
 	ctx.JSON(http.StatusOK, gin.H{
 		"message": message,
 	})
```

Comment:
message is confusing, don't you think?
when I am going through the code I will read it like:
`login returns a message'
but I am confused, what is message?
simimlary the API response also says `message`, it should be explicit like: 
access_token